project/cmake/CMakeLists.txt

```diff
@@ -24,7 +24,6 @@ option(ENABLE_PLIST       "Enable AirPlay support?" ON)
 option(ENABLE_NFS         "Enable NFS support?" ON)
 option(ENABLE_AIRTUNES    "Enable AirTunes support?" ON)
 option(ENABLE_CEC         "Enable CEC support?" ON)
-option(ENABLE_OPENSSL     "Enable OpenSSL?" ON)
```

Comment:
to be picky, we do use openssl on windows for curl and will be used for libssh as well, it's not needed for kodi build but it is needed for distribution. No clue if it's relevant, I just wanted to clarify.